public/app/features/alerting/unified/components/rules/RuleDetailsActionButtons.tsx

```diff
@@ -185,7 +185,7 @@ export const RuleDetailsActionButtons: FC<Props> = ({ rule, rulesSource }) => {
     );
   }
 
-  if (isEditable && rulerRule && !isFederated) {
+  if (isEditable && rulerRule && !isFederated && !isProvisioned) {
```

Comment:
Should we combine `isProvisioned` into `useIsRuleEditable` hook (either as a separate variable or evaluated as a part of `isEditable` and `isRemovable`)? If it's provisioned it's neither editable nor removable
When working on RBAC I had an idea that `useIsRuleEditable` could be a single source of truth about what you can do with an alert rule 😄

Reply:
I love this idea, unfortunately most of the information we need comes from different sources (some props, other from the redux store and some from `grafanaBootData`) so it might be a bit tricky to come up with a sensible hook.